config.style

```diff
@@ -1107,6 +1107,86 @@ Layer{
 	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/tn-ro/4.0/:RoadNode
 	Styles: TN_RO_RoadNode
 }
+Layer{
```

Comment:
This layer seems to be a mix from ObservedSoilProfile and SoilPlot - @Kate-Lyndegaard is that expected?

Reply:
@JohannaOtt Please check the specification:
![grafik](https://user-images.githubusercontent.com/36956461/133889158-19b52761-bfff-4508-8198-a2a015b2bb2d.png)